phpstan.neon.dist

```diff
@@ -0,0 +1,70 @@
+parameters:
+  level: 1
+  paths:
+    - src
+  scanFiles:
+    - wp-config.php
+    - src/wp-admin/includes/ms.php
+  scanDirectories:
+    - src/wp-includes
+    - src/wp-admin
+  excludes_analyse:
+    - src/wp-admin/includes/class-ftp.php
+    - src/wp-admin/includes/class-ftp-pure.php
+    - src/wp-admin/includes/class-ftp-sockets.php
+    - src/wp-admin/includes/class-pclzip.php
+    - src/wp-admin/includes/deprecated.php
+    - src/wp-admin/includes/noop.php
+    - src/wp-admin/includes/ms-deprecated.php
+    - src/wp-content
+    - src/wp-includes/atomlib.php
+    - src/wp-includes/class-json.php
+    - src/wp-includes/class-phpmailer.php
+    - src/wp-includes/class-requests.php
+    - src/wp-includes/class-simplepie.php
+    - src/wp-includes/class-smtp.php
+    - src/wp-includes/class-snoopy.php
+    - src/wp-includes/class-wp-http-ixr-client.php
+    - src/wp-includes/class-wp-http-requests-hooks.php
+    - src/wp-includes/class-wp-http-requests-response.php
+    - src/wp-includes/class-wp-simplepie-file.php
+    - src/wp-includes/class-wp-simplepie-sanitize-kses.php
+    - src/wp-includes/class-wp-text-diff-renderer-inline.php
+    - src/wp-includes/class-wp-text-diff-renderer-table.php
+    - src/wp-includes/class-wp-feed-cache.php
+    - src/wp-includes/deprecated.php
+    - src/wp-includes/ms-deprecated.php
+    - src/wp-includes/pluggable-deprecated.php
+    - src/wp-includes/rss.php
+    - src/wp-includes/ID3
+    - src/wp-includes/PHPMailer
+    - src/wp-includes/pomo
+    - src/wp-includes/random_compat
+    - src/wp-includes/Requests
+    - src/wp-includes/SimplePie
+    - src/wp-includes/sodium_compat
+    - src/wp-includes/Text
+  bootstrapFiles:
+    - tests/phpstan-bootstrap.php
```

Comment:
I think this might be better in it's own folder inside `/tests` rather than the root as currently `/tests` has all the different testing tools seperated out.